packages/neuron-wallet/src/controllers/multisig.ts

```diff
@@ -216,4 +217,12 @@ export default class MultisigController {
       }
     }
   }
+
+  async getMultisigBalances(isMainnet: boolean) {
+    const balances = await CellsService.getMultisigBalance(isMainnet)
```

Comment:
The outside method is named `getMultisigBalances` while the inner one is named `getMultisigBalance`